packages/react/src/side-nav/docs/overview.mdx

```diff
@@ -81,9 +81,118 @@ The background of the `SideNav` must match the background it is being placed on.
 <DoHeading />
 
 - include a concise label
-- indent each nesting level
+- indent each sub-level
 
 <DontHeading />
 
-- create more than 3 levels of nesting
-- place on the right-hand side of content.
+- create more than 3 sub-levels
+- place on the right-hand side of content
+- keep all lists with sub-levels open when there are many items.
+
+## Sub-level items
+
+To help users comprehend the side navigation, choose a strategy that is appropriate for the number of items in the list.
+
+### Visible when active
+
+When the list of items is long, prefer showing any sub-level lists when either navigating to the parent item or one of the sub-level items. Since this is the default behaviour, either setting `subLevelVisible="whenActive"` or omitting this prop will use this strategy.
+
+```jsx live
+<SideNav
+	title="Lodging your tax return"
+	titleLink="#"
+	collapseTitle="In this section"
+	activePath="#in-detail/record-keeping"
+	items={[
+		{
+			href: '#welcome',
+			label: 'Welcome',
+		},
+		{
+			href: '#do-you-need-to-lodge-a-tax-return',
+			label: 'Do you need to lodge a tax return?',
+		},
+		{
+			href: '#lodge-online',
+			label: 'Lodge online',
+			items: [
+				{
+					href: '#lodge-online/pre-filling',
+					label: 'Pre-filling your online tax return',
+				},
+			],
+		},
+		{
+			href: '#whats-new-for-individuals',
+			label: 'What’s new for individuals',
+		},
+		{
+			href: '#why-you-may-receive-a-tax-bill',
+			label: 'Why you may receive a tax bill',
+		},
+		{
+			href: '#in-detail',
+			label: 'In detail',
+			items: [
+				{
+					href: '#in-detail/record-keeping',
+					label: 'Record keeping',
+					items: [
+						{
+							href: '#in-detail/record-keeping/tax',
+							label: 'Keeping your tax records',
+						},
+						{
+							href: '#in-detail/record-keeping/incorrect-amounts',
+							label: 'Incorrect amounts',
+						},
+					],
+				},
+				{
+					href: '#tax-receipt',
+					label: 'Tax receipt',
+				},
+				{
+					href: '#pre-fill-availability',
+					label: 'Pre-fill availability',
+				},
+			],
+		},
+	]}
+	subLevelVisible="whenActive"
```

Comment:
I know this follows our standard pattern on new props at the end, but I wonder if moving this above `items` will help users understand (since I was also confused that first time)